src/test/java/com/tinkoff/edu/AppTest.java

```diff
@@ -5,48 +5,189 @@
 import com.tinkoff.edu.app.LoanCalcController;
 import com.tinkoff.edu.app.LoanCalcRepository;
 import com.tinkoff.edu.app.LoanRequest;
+import com.tinkoff.edu.app.LoanRequestType;
 import com.tinkoff.edu.app.LoanResponse;
 import com.tinkoff.edu.app.LoanResponseType;
-import com.tinkoff.edu.app.LoanType;
 import com.tinkoff.edu.app.VariableLoanCalcRepository;
 import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.fail;
+
 public class AppTest {
     private LoanRequest loanRequest;
     private LoanCalcController controller;
     private LoanCalcRepository repository;
 
-    @BeforeEach
-    public void init() {
-        loanRequest = new LoanRequest(10, 1000, LoanType.IP);
+
+    @Test
+    @DisplayName("Проверка одобрения заявки: loanType = PERSON, amount <= 10000, months <= 12")
+    public void checkApproveRequestForPerson1() {
+        loanRequest = new LoanRequest(12, 10000, LoanRequestType.PERSON);
+        repository = new VariableLoanCalcRepository();
+        controller = new LoanCalcController(new CalculationsWithAllParamsService(repository));
+        LoanResponse loanResponse = controller.createRequest(loanRequest);
+        System.out.println("Your request number is: " + loanResponse.getRequestId() + "; "
+                + "Your request status is: " + loanResponse.getIsAccepted());
+        finalCheck(LoanResponseType.APPROVED, loanResponse);
+    }
+
+    @Test
+    @DisplayName("Проверка одобрения заявки: loanType = PERSON, amount <= 10000, months > 12")
+    public void checkApproveRequestForPerson2() {
+        loanRequest = new LoanRequest(14, 10000, LoanRequestType.PERSON);
+        repository = new VariableLoanCalcRepository();
+        controller = new LoanCalcController(new CalculationsWithAllParamsService(repository));
+        LoanResponse loanResponse = controller.createRequest(loanRequest);
+        System.out.println("Your request number is: " + loanResponse.getRequestId() + "; "
+                + "Your request status is: " + loanResponse.getIsAccepted());
+        finalCheck(LoanResponseType.DECLINED, loanResponse);
+    }
+
+    @Test
+    @DisplayName("Проверка одобрения заявки: loanType = PERSON, amount > 10000, months > 12")
```

Comment:
может сбить с толку - лучше явно указать ожидаемый результат - отклонение
```suggestion
    @DisplayName("Проверка отклонения заявки: loanType = PERSON, amount > 10000, months > 12")
```